docs/2.0/docs/accountfactory/architecture/security-controls.md

```diff
@@ -0,0 +1,231 @@
+# Controls
+
+Gruntwork Account Factory employs a defense-in-depth approach to secure workflows across both GitHub and GitLab platforms. This document outlines the controls Pipelines uses to ensure that only infrastructure written in code and approved by a reviewer can be deployed to your AWS accounts.
```

Comment:
It's a bit jarring to say "Gruntwork Account Factory employs a defense-in-depth approach..." and then "This document outlines the controls _Pipelines_ uses...".

If we're saying that all the security for Account Factory depends on Pipelines, we should call out that connection explicitly, and then highlight that this doc will focus on which controls in Pipelines enable security for Account Factory.